templates/ep19/bs/cfp/preview.html

```diff
@@ -0,0 +1,75 @@
+{% extends "ep19/bs/base.html" %}
+
+{% load crispy_forms_tags %}
+
+
+{% block content %}
+
+<div class="container page" id="cfp_page">
+    <div class="row">
+        <div class="col-md-12">
+            <h1>{{ talk_as_dict.title }}</h1>
+            <h3>{{ talk_as_dict.subtitle }}</h3>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-8">
+            <div>
+                Type: {{ talk_as_dict.type_display }}<br>
+                Python level: {{ talk_as_dict.python_level }}<br>
+                Domain level: {{ talk_as_dict.domain_level }}<br>
+                Tags: 
+                {% for tag in talk_as_dict.tags %}
+                <span class='btn btn-outline-info'>{{ tag }}</span>
+                {% endfor %}
+            </div>
+            <hr>
+            <div>
+                {{ talk_as_dict.abstract_short|linebreaks }}
+            </div>
+            <hr>
+            <div>
+                {{ talk_as_dict.abstract|linebreaks }}
+            </div>
+            <hr>
+            <div>
+                {{ talk_as_dict.abstract_extra|linebreaks }}
+            </div>
+            <hr>
+
+            <div style='margin-top:5em'>
+                {# TODO: this should be possible only if CFP is still open #}
```

Comment:
Is this todo not handled below? The condition seems to check for the CFP status.